### DISCUSSION TITLE
Manual Mod Configuration Numbers For osu!lazer

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#28648] [Ideas] [answered]
I have been believing that the mod configuration menu for osu!lazer just being sliders has been quite a bit hard to get a specific number, especially when enabling "Extended limits" in Difficulty Adjust or in Double Time/Daycore mods having speeds as specific to the hundredth of increments.

**Visual Apprearacne**: When next to a slider, adding text next to the right of the slider, when clicked on any number will be able to edit the number with a darker box surrounding the selected editing number. When not editing any number it will show as only a number with no any darker box around the number.

**How it works**: When there are any sliders to edit the value of editing a mod, there will be a number next to the slider to the right, when done manually entering a number will make the number to the specified number that was inputted in. For example after manually entering a number of speed within Double Time, such as 1.30, the slider and the number will get set to 1.30. 

This is **Before**
![Screenshot 2024-06-27 151256](https://github.com/ppy/osu/assets/171513312/aa09bcef-7ad3-49dd-9f29-c0711bb9399f)

This is **After**
![Untitled107_20240627152740](https://github.com/ppy/osu/assets/171513312/428d49ed-6029-4765-929f-4d37c1449910)
peppy or his development team can make something similar or completely remake this idea.

**Comment by Tenexxt** [comment]
This would be actually usefull for all mods with an option to customize them, like half time, double time, and **ESPECIALLY** on accuracy challenge (or basiclly any mod custom with a slider in it)

**Comment by Stoppedpuma** [comment, marked as answer]
Duplicate of #24900
